core/src/builtin_widgets.rs

```diff
@@ -709,6 +709,20 @@ impl<T> FatObj<T> {
     on_mixin!(self, on_focus_out, f)
   }
 
+  /// Attaches a handler to the specific custom event that is bubbled from the
+  /// descendants.
+  pub fn on_custom_event<E: 'static>(
```

Comment:
How about renaming these two methods as on_custom_event and on_custom_event_of<E>?

Reply:
or `on_concrete_custom_event`